Paper.tex

```diff
@@ -413,17 +413,45 @@ \subsection{The Transaction} \label{subsec:transaction}
 \mathbb{B}_{0} & \text{otherwise}\end{cases}
 \end{equation}
 
+\subsection{The Withdrawal}\linkdest{withdrawal}\label{subsec:The_Withdrawal}
+A withdrawal (formally, \textit{W}) is a tuple of data describing a consensus layer validator's withdrawal of some amount of its staked Ether.
+A withdrawal is created and validated in the consensus layer of the blockchain and then pushed to the execution layer.
+A withdrawal is composed of the following fields:
+
+\begin{description}
+\item[globalIndex] zero based incrementing withdrawal index that acts as a unique identifier for this withdrawal; formally $W_{\mathrm{g}}$.
+\item[validatorIndex] index of consensus layer's validator this withdrawal corresponds to; formally $W_{\mathrm{v}}$.
+\item[recipient] the 20-bytes address that will receives Ether form this withdrawal; formally $W_{\mathrm{r}}$.
+\item[amount] a nonzero amount of Ether denominated in Gwei ($10^9$ Wei); formally $W_{\mathrm{a}}$.
```

Comment:
`Gwei` could just go as a definition in section `2.1.`

Reply:
Good catch! Since `EIP-4895` explicitly define this value as being denominated in `Gwei`, I think you are right.

Reply:
changes in [55a4099](https://github.com/ethereum/yellowpaper/pull/906/commits/55a4099d6366a2425e4c7c8ad1ba2687cd1abc89)